feature_engine/encoding/base_encoder.py

```diff
@@ -2,28 +2,22 @@
 from typing import List, Optional, Union
 
 import pandas as pd
-from sklearn.base import BaseEstimator, TransformerMixin
-from sklearn.utils.validation import check_is_fitted
-
 from feature_engine._docstrings.methods import _get_feature_names_out_docstring
 from feature_engine._docstrings.substitute import Substitution
-from feature_engine.dataframe_checks import (
-    _check_contains_na,
-    _check_X_matches_training_df,
-    check_X,
-)
-from feature_engine.encoding._docstrings import (
-    _errors_docstring,
-    _ignore_format_docstring,
-    _variables_docstring,
-)
+from feature_engine.dataframe_checks import (_check_contains_na,
```

Comment:
can we revert to the original indentation please? so it is coherent with the rest of the codebase?